internal/service/osis/pipeline_test.go

```diff
@@ -230,6 +230,9 @@ func TestAccOpenSearchIngestionPipeline_vpc(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "vpc_options.0.security_group_ids.0"),
 					resource.TestCheckResourceAttr(resourceName, "vpc_options.0.subnet_ids.#", acctest.Ct1),
 					resource.TestCheckResourceAttrSet(resourceName, "vpc_options.0.subnet_ids.0"),
+					resource.TestCheckResourceAttr(resourceName, "vpc_options.0.vpc_attachment_options.#", acctest.Ct1),
+					resource.TestCheckResourceAttrSet(resourceName, "vpc_options.0.vpc_attachment_options.0"),
+					resource.TestCheckResourceAttr(resourceName, "vpc_options.0.vpc_endpoint_management", "CUSTOMER"),
```

Comment:
would someone be able to guide me with these tests? i'm not sure if what i did here is correct / sufficient